src/admin-views/troubleshooting/footer.php

```diff
@@ -0,0 +1,14 @@
+<?php 
+/**
+ * View: Troubleshooting - Footer
+ * 
+ * @since TBD
+ * 
+ */
+?>
+    <img
+        class="tribe-events-admin-footer-logo"
+        src="<?php echo esc_url(tribe_resource_url('images/logo/tec-brand.svg', false, null, $main)); ?>"
+        alt="<?php esc_attr_e('The Events Calendar brand logo', 'tribe-common'); ?>"
+    />
+</div>
```

Comment:
You have a stray `</div>` here, was there supposed to be an opening div? Also if this a footer, let's use the `<footer>` element.

Reply:
Its not a stray div, I've added a comment to the file to describe where its coming from